cfgov/v1/management/commands/delete_page.py

```diff
@@ -0,0 +1,43 @@
+from django.core.exceptions import MultipleObjectsReturned
+from django.core.management.base import BaseCommand, CommandError
+
+from v1.models import CFGOVPage
+from v1.wagtail_hooks import flush_akamai
+
+
+class Command(BaseCommand):
+    help = 'Deletes a page by its slug name'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--slug',
+            nargs='?',
+            help='The slug for the page you wish to delete'
+        )
+        parser.add_argument(
+            '--id',
+            nargs='?',
+            help='The ID for the page you wish to delete'
+        )
+
+    def handle(self, *args, **options):
+        page_to_delete = None
+        slug = options['slug']
+        page_id = options['id']
+        if slug:
+            try:
+                page_to_delete = CFGOVPage.objects.get(slug=slug)
+            except MultipleObjectsReturned as e:
```

Comment:
I think I'd still like to see path resolution sometime in the future, but that's not a blocker right now. Let's see how we end up using it.